src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -410,202 +673,209 @@ export const gravityStitchingEnvironment = (
+      },
+      Query: {
+        curatedMarketingCollections: {
           fragment: gql`
-            ... on ArtistSeries {
-              forSaleArtworksCount
-              artworksCount
+            ...on Query {
+              __typename
             }
           `,
-          resolve: async ({ forSaleArtworksCount, artworksCount }) => {
-            let artworksCountMessage
-
-            if (forSaleArtworksCount) {
-              artworksCountMessage = `${forSaleArtworksCount} available`
-            } else {
-              artworksCountMessage = `${artworksCount} ${
-                artworksCount === 1 ? "work" : "works"
-              }`
+          resolve: async (_parent, args, context, info) => {
+            try {
+              return await info.mergeInfo.delegateToSchema({
+                schema: gravitySchema,
+                operation: "query",
+                fieldName: "marketingCollections",
+                args: {
+                  slugs: [
+                    "trending-now",
+                    "top-auction-lots",
+                    "new-this-week",
+                    "curators-picks-blue-chip",
+                    "curators-picks-emerging",
+                  ],
+                  ...args,
+                },
+                context,
+                info,
+              })
+            } catch (error) {
+              return []
             }
-
-            return artworksCountMessage
           },
         },
-        image: {
+      },
+      SearchCriteria: {
+        artistsConnection: {
           fragment: gql`
-          ... on ArtistSeries {
-            image_url: imageURL
-            original_height: imageHeight
-            original_width: imageWidth
-            representativeArtworkID
-          }
-          `,
-          resolve: async (
-            {
-              representativeArtworkID,
-              image_url,
-              original_height,
-              original_width,
-            },
-            args,
-            context,
-            info
-          ) => {
-            let imageData: unknown
-            if (image_url) {
-              imageData = {
-                image_url,
-                original_width,
-                original_height,
-              }
-            } else if (representativeArtworkID) {
-              const { artworkLoader } = context
-              const { images } = await artworkLoader(representativeArtworkID)
-              imageData = normalizeImageData(getDefault(images))
+            ... on SearchCriteria {
+              artistIDs
             }
-
+          `,
+          resolve: ({ artistIDs }, args, context, info) => {
             return info.mergeInfo.delegateToSchema({
-              args,
               schema: localSchema,
               operation: "query",
-              fieldName: "_do_not_use_image",
-              context: {
-                ...context,
-                imageData,
+              fieldName: "artistsConnection",
```

Comment:
And here we're adding `artistsConnection` to the `SearchCriteria` type, which resolves `artistIDs` data in a nice way that we can use and traverse in the graph.